lib/benchee/conversion/scale.ex

```diff
@@ -12,6 +12,7 @@ defmodule Benchee.Conversion.Scale do
   @type unit_atom :: atom
   @type any_unit :: unit | unit_atom
   @type scaled_number :: {number, unit}
+  @type best_unit_strategy :: :best | :largest | :smallest | :none
```

Comment:
This is super nit-picky, but do you think just `unit_strategy` might be a better name for this type? We can be referencing any of these strategies without any one of them being the best. It might also be confusing that one of the `best_unit_stragegies` is itself `:best`.

What do you think? I don't have strong feelings about this, but it jumped out to me as kind of strange when I read it.

Reply:
I'm all about nit picky code reviews :+1: 

the strategy is about how the "best fit unit" for a set of values is chosen to scale them to. so the duplication of `best` is unfortunate but not exactly the same... the method is also called `best_unit` and that's the strategy for `best_unit`. Not sure what `unit_strategy` conveys... might be enough as I also thought about renaming it that way :)

Reply:
what about `unit_scaling_strategy` or `scaling_strategy`?

Reply:
I guess we can go with `scaling_strategy` - will make the adjustments, thanks for the discussion :green_heart: